geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/kafka/commands/KeywordCommand.scala

```diff
@@ -0,0 +1,105 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.kafka.commands
+
+import com.beust.jcommander._
+import com.beust.jcommander.converters.IParameterSplitter
+import org.locationtech.geomesa.kafka.{KafkaDataStore, KafkaDataStoreSchemaManager}
+import org.locationtech.geomesa.tools.kafka.commands.KeywordCommand.KeywordParameters
+import org.locationtech.geomesa.tools.common.OptionalFeatureTypeNameParam
+import org.locationtech.geomesa.tools.kafka.ProducerKDSConnectionParams
+import scala.collection.JavaConversions._
+import scala.io.StdIn
+import scala.util.{Try, Failure}
+
+class KeywordCommand(parent: JCommander) extends CommandWithKDS(parent) {
+
+  import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType._
+  override val command: String = "keyword"
+  override val params = new KeywordParameters()
+
+  override def execute(): Unit = {
+
+    if (ds == null) {
+      throw new ParameterException("Could not load a data store with the provided parameters")
+    }
+
+    val sft = ds.asInstanceOf[KafkaDataStore].getKafkaSchema(params.featureName)
+    var keywordsModified = false
+
+    if (params.keywordsToAdd != null) {
+      sft.addKeywords(params.keywordsToAdd.mkString(KEYWORDS_DELIMITER))
+      keywordsModified = true
+    }
+
+    if (params.keywordsToRemove != null) {
+      sft.removeKeywords(params.keywordsToRemove.mkString(KEYWORDS_DELIMITER))
+      keywordsModified = true
+    }
+
+    if (params.removeAll) {
+      val confirm = StdIn.readLine("Remove all keywords? (y/n): ").toLowerCase
+      if (confirm.equals("y") || confirm.equals("yes")) {
+        sft.removeAllKeywords()
+        keywordsModified = true
+      } else {
+        println("Aborting operation")
+        ds.dispose()
+        return
+      }
+    }
+
+    // Attempt to replace the old schema with the updated one
+    if (keywordsModified) {
+      Try {
+        ds.removeSchema(sft.getTypeName)
+      } match {
+        case Failure(ex) => println(ex.toString)
```

Comment:
I decided to continue because I would run into a "NodeExists" exception even though the schema is deleted correctly. I noticed this exception also happens with the RemoveSchema command [here](https://github.com/locationtech/geomesa/blob/77a676462b99f060b178aa27e055709a455c3c25/geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/RemoveSchemaCommand.scala#L54) 
Though I should at least match that specific exception